src/dmd/hdrgen.d

```diff
@@ -3433,3 +3438,24 @@ extern (C++) const(char)* parametersTypeToChars(Parameters* parameters, int vara
     v.parametersToBuffer(parameters, varargs);
     return buf.extractString();
 }
+
+/** Pretty print function parameters.
```

Comment:
This has the same description and return value as the previous function - what does it do different?

Reply:
Typo, should be *parameter*. Fixed.